huma.go

```diff
@@ -582,8 +582,11 @@ func Register[I, O any](api API, op Operation, handler func(context.Context, *I)
 			if c := f.Tag.Get("contentType"); c != "" {
 				contentType = c
 			}
-
-			s := SchemaFromField(registry, f, getHint(inputType, f.Name, op.OperationID+"Request"))
+			hint := getHint(inputType, f.Name, op.OperationID+"Request")
+			if nameHint := f.Tag.Get("name-hint"); nameHint != "" {
```

Comment:
One small nit: our existing tags (e.g. see https://huma.rocks/features/request-validation/#validation-tags) use `lowerCamelCasing` so we should probably use `nameHint` for consistency.

Reply:
Alright !  Changed it to `nameHint`